setup.py

```diff
@@ -86,7 +86,7 @@ def read_version():
             "pytest-cov",
             "mock",
             "sagemaker[local]>=1.16.2",
-            "black==19.3b0 ; python_version >= '3.6'",
+            "black==19.3b0 ; python_version >= '3.7'",
```

Comment:
I think this should probably stay 3.6 - the purpose is to prevent it from trying to install black for Python 2